go.mod

```diff
@@ -397,3 +397,5 @@ require (
 	github.com/ziutek/mymysql v1.5.4 // indirect
 	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
 )
+
+replace go.viam.com/api => ../api
```

Comment:
To use https://github.com/viamrobotics/api/pull/430. Hold off on reviewing this until those proto changes are merged.